lib/utils.ts

```diff
@@ -18,3 +18,10 @@ export function getAvatarInitials(fullname?: string): string {
 
   return initials.toUpperCase();
 }
+
+export function getFileSize(size?: number): string {
+  if (!size) return "";
+  if (size < 1000) return `${size} KB`;
```

Comment:
This looks incorrect, maybe we can use https://www.npmjs.com/package/pretty-bytes instead :D